tools/wptserve/tests/functional/test_request.py

```diff
@@ -125,13 +124,13 @@ def handler(request, response):
         self.server.router.register(*route)
 
         # Try some non-ASCII characters and the server shouldn't crash.
-        encoded_text = u"你好".encode("utf-8")
+        encoded_text = "你好".encode()
```

Comment:
Unconvinced by this, hmm?

Reply:
It's equivalent per https://docs.python.org/3/library/stdtypes.html#str.encode, but maybe you mean being explicit is better?

Reply:
I'll revert all changes like this.